### PR TITLE
Convert semantics_node from SkMatrix44 to SkM44

### PR DESCRIPTION
**Pull request by brianosman**

### CODE REVIEW

File: shell/platform/fuchsia/flutter/accessibility_bridge.cc [hunk 1]
```diff
@@ -282,22 +282,12 @@ void AccessibilityBridge::UpdateScreenRects(
...
+  SkV4 dst[2] = {
+      current_transform.map(rect.left(), rect.top(), 0, 1),
+      current_transform.map(rect.right(), rect.bottom(), 0, 1),
   };
-  SkScalar dst[4 * 4];
-  current_transform.map2(quad, 4, dst);
-  node.screen_rect.setLTRB(dst[0], dst[1], dst[8], dst[9]);
+  node.screen_rect.setLTRB(dst[0].x, dst[0].y, dst[1].x, dst[1].y);
   node.screen_rect.sort();
-  std::vector<SkVector4> points = {
```
Comment (chinmaygarde) [inline shell/platform/fuchsia/flutter/accessibility_bridge.cc:295]: huh, this was just unused?

Comment (brianosman) [reply shell/platform/fuchsia/flutter/accessibility_bridge.cc:295]: Yup, it was sort of redundant with the previous chunk of code, and then the result was never used. Also, the previous code transformed all four corners, but then only used the results from two of them.

**Comment by aam** [comment]
Not sure if it was intentional, but I don't think we generally want to land CLs with failed checks, on red tree unless those CLs are directed at turning the tree green.

**Comment by brianosman** [comment]
Sorry about this. This is part of my ongoing frustration with the checks system here. I re-tried the two red checks, and they both passed, but the UI here never picks up that information.

**Pull request merged**